src/paperqa/clients/semantic_scholar.py

```diff
@@ -40,7 +40,8 @@
     "volume": {"journal"},
     "pages": {"journal"},
     "journal": {"journal"},
-    "url": {"url", "openAccessPdf"},
+    "url": {"url"},
+    "pdf_url": {"openAccessPdf"},
```

Comment:
Since you've acquired some insight into `pdf_url` vs `url` here, do you mind adding a `description` to the `DocDetails.pdf_url` field just documenting its role.

And feel free to ignore this, just it's also been a point of confusion for me